### PR TITLE
feat: Add Running... note to the init command

### PR DESCRIPTION
**Pull request by cortinico**
Summary:
---------

This is an implementation part of https://github.com/react-native-community/discussions-and-proposals/pull/759
I'm waiting for the RFC to be approved before we can merge this, but I'd like to get a review.

Test Plan:
----------

@szymonrybczak do you know how I can easily test this?

Checklist
----------

- [x] Documentation is up to date to reflect these changes.
- [x] Follows commit message convention described in [CONTRIBUTING.md](https://github.com/react-native-community/cli/blob/main/CONTRIBUTING.md#commit-message-convention)

### CODE REVIEW

File: packages/cli/src/commands/init/init.ts [hunk 1]
```diff
@@ -204,6 +204,13 @@ async function createFromTemplate({
   yarnConfigOptions,
 }: TemplateOptions): Promise<TemplateReturnType> {
   logger.debug('Initializing new project');
+
+  // We print this info notice as per https://github.com/react-native-community/discussions-and-proposals/pull/759
+  logger.log(`
```
Comment (thymikee) [inline packages/cli/src/commands/init/init.ts:209]:
```suggestion
  logger.info(`
```

**Review by thymikee — COMMENTED** [review]
You can check our e2e tests under `__e2e__/init.test.ts`. Some will likely fail because this will be logged in the text stream written to snapshots

**Comment by cortinico** [comment]
> You can check our e2e tests under `__e2e__/init.test.ts`. Some will likely fail because this will be logged in the text stream written to snapshots

Great will wait for test failures to follow up here then

**Comment by cortinico** [comment]
Closing in favor of https://github.com/facebook/react-native/pull/43127

**Pull request closed without merging**